src/features/metadata/components/EnvBuildStatus.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { CircularProgress, Typography } from "@mui/material";
+import Link from "@mui/material/Link";
+import OpenInNewIcon from "@mui/icons-material/OpenInNew";
+import { Artifact, Build } from "../../../common/models";
+import { PrefContext } from "../../../preferences";
+import { StyledMetadataItem } from "../../../styles/StyledMetadataItem";
+import artifactList from "../../../utils/helpers/artifact";
+import { artifactBaseUrl } from "../../../utils/helpers/parseArtifactList";
+import { buildStatus } from "../../../utils/helpers/buildMapper";
+
+const LogLink = ({ logArtifact }: { logArtifact: Artifact }) => {
+  const pref = React.useContext(PrefContext);
+  const url = new URL(
+    logArtifact.route,
+    artifactBaseUrl(pref.apiUrl, window.location.origin)
+  );
+  return (
+    <Link
+      href={url.toString()}
+      target="_blank"
+      sx={{
+        display: "inline-flex",
+        verticalAlign: "bottom", // align link (icon plus link text) with non-link text on the same line
+        alignItems: "center" // align icon and text within link
+      }}
+    >
+      <OpenInNewIcon sx={{ mr: 0.5 }} fontSize="inherit" />
+      Log
+    </Link>
+  );
+};
+
+export interface IEnvBuildStatusProps {
```

Comment:
What's the usual protocol around exporting these one-off interfaces? I know some projects do so and others don't.